src/commands/utility/changeStatus.js

```diff
@@ -1,96 +1,77 @@
-  const status = options.getString('status');
-  const message = options.getString('message');
-  const newStatus = MAPPED_STATUS_COMMANDS[status];
-
-  if (!newStatus) {
-    return await interaction.editReply(
-      translateLanguage('changeStatus.invalidStatus')
+const createStatusMenu = (category) => {
+  return new StringSelectMenuBuilder()
+    .setCustomId('status_select')
+    .setPlaceholder('Make a selection!')
+    .addOptions(
+      Object.keys(MAPPED_STATUS_COMMANDS[category]).map((status) => {
+        return new StringSelectMenuOptionBuilder()
+          .setLabel(status)
+          .setDescription(status)
+          .setValue(status);
+      })
     );
-  }
-
-  await setThreadNameWithStatus({ channel, newStatus });
-
-  if (message) {
-    const markdownMessage =
-      `# ${newStatus} ${status.replaceAll('-', ' ')}\n\n` +
-      `${message}\n\n` +
-      `> ${user}`;
-    await channel.send(markdownMessage);
-  }
-
-  await interaction.editReply(
-    translateLanguage('changeStatus.updatedStatus', {
-      status: status.replaceAll('-', ' '),
-    })
-  );
 };
 
 module.exports = {
   data: new SlashCommandBuilder()
     .setName('change-status')
     .setDescription(translateLanguage('changeStatus.description'))
-    .setDescriptionLocalizations(keyTranslations('changeStatus.description'))
-    .addStringOption((option) =>
-      option
-        .setName('status')
-        .setDescription(translateLanguage('changeStatus.statusOption'))
-        .setDescriptionLocalizations(
-          keyTranslations('changeStatus.statusOption')
-        )
-        .setRequired(true)
-        .addChoices(
-          COMMAND_KEYS.map((command) => ({
-            name: command.replaceAll('pr-', '').replaceAll('-', ' '),
-            value: command,
-          }))
-        )
-    )
-    .addStringOption((option) =>
-      option
-        .setName('message')
-        .setDescription(translateLanguage('changeStatus.messageOption'))
-        .setDescriptionLocalizations(
-          keyTranslations('changeStatus.messageOption')
-        )
-        .setRequired(false)
-    ),
+    .setDescriptionLocalizations(keyTranslations('changeStatus.description')),
   async execute(interaction) {
     try {
-      await interaction.deferReply({ ephemeral: true });
+      const { channel } = interaction;
+      let selectMenu;
 
-      if (!interaction.channel.isThread()) {
-        return await interaction.editReply({
-          content: translateLanguage('changeStatus.notAThread'),
-          ephemeral: true,
-        });
+      switch (channel.name) {
+        case 'novabot':
+          selectMenu = createStatusMenu('novabot');
+          break;
+        case 'i18n-populator':
+          selectMenu = createStatusMenu('i18n-populator');
+          break;
+        case 'evo-crypter':
+          selectMenu = createStatusMenu('evo-crypter');
+          break;
+        default:
+          selectMenu = createStatusMenu('evo-crypter');
+          break;
       }
```

Comment:
What about passing channel.name instead?
`selectMenu = createStatusMenu(channel.name);`

Reply:
Maybe we would need a validation before to comprobate the channel.name is one of the available options